src/__test__/redux/actions/cellSets/__snapshots__/runCellSetsClustering.test.js.snap

```diff
@@ -6,17 +6,21 @@ Object {
 }
 `;
 
-exports[`runCellSetsClustering action Dispatches all required actions to update cell sets clustering. 2`] = `undefined`;
+exports[`runCellSetsClustering action Dispatches all required actions to update cell sets clustering. 2`] = `
+Object {
+  "type": "cellSets/clusteringUpdated",
+}
+`;
```

Comment:
how come is this not returning undefined now?

Reply:
`runCellSetsClustering` used to have a callback in its dispatch to `fetchWork`: https://github.com/hms-dbmi-cellenics/ui/blob/master/src/redux/actions/cellSets/runCellSetsClustering.js#L41-L47. This callback runs `updateCellSetsClustering` which dispatches the action.

This PR removes that callback and move its implementation to https://github.com/hms-dbmi-cellenics/ui/blob/0bcde96857db3fbb8a8f5798a8a17de8ca6a0002/src/utils/experimentUpdatesHandler.js#L70. Because `updateCellSetsClustering` is not called as a callback anymore, the action is not logged in the store. Hence it is `undefined`.